src/daemon/dpi.c

```diff
@@ -175,6 +176,9 @@ int updatedpi(usbdevice* kb, int force){
         data_pkt[6] = (newdpi->x[i] >> 8) & 0xFF;
         data_pkt[7] = newdpi->y[i] & 0xFF;
         data_pkt[8] = (newdpi->y[i] >> 8) & 0xFF;
+        data_pkt[9] = light->r[LED_MOUSE + N_MOUSE_ZONES + i];
+        data_pkt[10] = light->g[LED_MOUSE + N_MOUSE_ZONES + i];
+        data_pkt[11] = light->b[LED_MOUSE + N_MOUSE_ZONES + i];
```

Comment:
This modifies the packets for *all* mice, not just the Dark Core. In the spirit of trying not to break what presently works, could you wrap this in an `if (IS_DARK_CORE(kb))` please? I don't trust Corsair enough to assume that adding these packets won't break something.

Reply:
Sure thing.